rust/rsc/postgres-cmd-spec.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Can this be deleted if we are generated the spec file?

Reply:
yep, I just haven't gotten around to it, hold your horses on clean up comments.